### DISCUSSION TITLE
Welcome to Reservation2 Discussions!

### DISCUSSION THREAD

**Discussion by shibatea** [ModelingKai/Reservation2#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

---

私たちは、コミュニティの他のメンバーとつながる場所としてディスカッションを使用しています。ぜひご利用ください。

- 気になることを質問してください。
- アイデアを共有してください。
- 他のコミュニティメンバーとの交流
- 他のメンバーを歓迎し、心を開いてください。ここは私たちのコミュニティであることを忘れないでください。
- 一緒に作りましょう 💪.

始めたい方は、以下に自己紹介と、このコミュニティで何をしているのかをコメントしてください。

**Comment by jnuank** [comment]
書き込んでみる。

話すテーマを「New Discussion」で作ってから、ここで話し合えばいいんですね
